INSTALL.md

```diff
@@ -30,15 +30,26 @@ anywhere because Rizin is compiled in a "portable" way, allowing moving the
 whole directory anywhere.
 
 ## MacOS
-On MacOS systems, Rizin can be installed through .pkg files attached to the
+On MacOS systems you can install Rizin in one of three ways.
+
+### Homebrew
+Rizin offers a Homebrew [formula](https://formulae.brew.sh/formula/rizin)
```

Comment:
both here and in the next section i feel we should be more explicit on the fact that it is not us who provide these things, but they are maintained by the respective communities (homebrew and macports). We might help them keep them updated or fix things here and there, but we are not actively keeping them updated.

Reply:
I added some information tackling this topic. What do you think?

Reply:
LGTM